opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/TracingFilterTest.java

```diff
@@ -244,6 +245,23 @@ public void testAsyncImmediateExit() throws IOException {
         Assert.assertEquals("java-web-servlet", mockSpan.tags().get(Tags.COMPONENT.getKey()));
     }
 
+    @Test
+    public void testCurrentSpanRequest() throws IOException {
+        OkHttpClient client = new OkHttpClient();
+        Request request = new Request.Builder()
+                .url(localRequestUrl("/currentSpan"))
+                .build();
+
+        client.newCall(request).execute();
+
+        List<MockSpan> mockSpans = mockTracer.finishedSpans();
```

Comment:
One thing, the wait statement is missing before this. It can potentially cause unstable CI.